lib/team.rb

```diff
@@ -0,0 +1,9 @@
+class Team
+
+attr_reader :team_name, :team_id
+
+  def initialize(team)
+    @team_name = team[:teamname]
```

Comment:
underscore in :team_name

Reply:
That is how it get the data from the CSV. Took a long time to figure out that was the case. Good catch though!